functions/api/getLike.js

```diff
@@ -0,0 +1,47 @@
+import Sequelize from 'sequelize';
+import Like from './db/models/like';
+import db from './config/db';
+
+export async function main(event, context, callback) {
```

Comment:
Same thing here. I don't think we need this API.

Reply:
This can probably also be integrated to the comment API

The purpose of this API is to enable a user to Like/Unlike a comment. 
Not to actually fetch the comment. Just to determine if a user has already liked a specific comment, or not. 

There will be one single button for 'liking/unliking' a comment. 
This button will toggle between 'Like' and 'Unlike'.

To enable this toggle, for the logged in user, on a specific comment: 
API returns Null if the user has not liked the comment yet  
--> the user can then Like the comment (createLike)

API returns a comment if the user has already liked the comment 
--> the user can then Unlike the comment (deleteLike)

could just be a simple true/false